.circleci/config.yml

```diff
@@ -137,6 +137,36 @@ commands:
             # Enable routing from this service to the backend applications which are private
             cf add-network-policy "$CF_APP-$REVIEW_URL_ID" "$CF_BACKEND_APP_XI-dev" --protocol tcp --port 8080
             cf add-network-policy "$CF_APP-$REVIEW_URL_ID" "$CF_BACKEND_APP_UK-dev" --protocol tcp --port 8080
+      - run:
+          name: Find JIRA ticket number
+          command: |
+            REPO_API_URL=$(echo "${CIRCLE_REPOSITORY_URL}/" | sed 's|.git/$||' | sed 's|git@github.com:|https://api.github.com/repos/|')
+            JIRA_TICKET=$(curl --silent --show-error "${REPO_API_URL}/pulls/${PR_NUMBER}" | jq -r '.body' | grep -oP 'HOTT-\d{4,5}' | head -1 || true)
```

Comment:
Is this HOTT- regex a millenium bug?

Reply:
Yes in theory - though if it covers up to just 1 less than a 100 thousand tickets so I can't see it being a problem